variables.tf

```diff
@@ -0,0 +1,99 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+###-----------------------------------------------------------------------------
+### general config
+###-----------------------------------------------------------------------------
+
+variable "project_name" {
+  type        = string
+  description = "the base name to use when creating resources. a randomized suffix will be added."
+  default     = "gcp-meanstack-demo"
+}
+
+###-----------------------------------------------------------------------------
+### region config
+###-----------------------------------------------------------------------------
+
+# Please refer to https://www.mongodb.com/docs/atlas/reference/google-gcp/#std-label-google-gcp
+# for a mapping of Atlas region names to Google Cloud region names. In most cases
+# you should use the same region for both services.
+
+variable "google_cloud_region" {
+  type        = string
+  description = "the Google Cloud region in which to create resources"
+  default     = "us-central1"
+}
+
+variable "atlas_cluster_region" {
+  type        = string
+  description = "the Atlas region in which to create the database cluster"
+  default     = "CENTRAL_US"
+}
+
+variable "app_image" {
+  type        = string
+  description = "the fully-qualified name of your app image"
+  default     = "us-central1-docker.pkg.dev/next22-mean-stack-demo/demo-app/server:latest"
```

Comment:
Maybe add a note in the Readme that this can be any image as long as it has an application which can read a MongoDB URI from an environment variable and interact with a database. Someone might want to sub in their own stuff to see how it works.

Reply:
That's a great point. I was thinking about adding a "next steps" section at the bottom of the readme with some ideas of ways that people can experiment. Would adding that section and a blurb about trying different images work for you?

Reply:
added in 0b3f65c, let me know what you think

Reply:
beautiful! That's perfect.